terraform/install/main.tf

```
@@ -10,6 +10,6 @@ resource "null_resource" "nixos-remote" {
     environment = {
       SSH_PRIVATE_KEY = var.ssh_private_key
     }
-    command = "nix run --extra-experimental-features 'nix-command flakes' ${path.module}#nixos-anywhere -- ${local.nixos_anywhere_flags}"
+    command = "nix run --extra-experimental-features 'nix-command flakes' path:$(readlink -f ${path.module})/../..#nixos-anywhere -- ${local.nixos_anywhere_flags}"
```

```suggestion
    command = "nix run --extra-experimental-features 'nix-command flakes' path:${dirname(dirname(abspath(path.module)))})#nixos-anywhere -- ${local.nixos_anywhere_flags}"
```
`readlink -f` is not available on macOS's BSD readlink

---

Looks like readlink was needed after all.

---

`path:` seems to fix detection of the flake.